README.md~

```diff
@@ -0,0 +1,25 @@
+This is a place to practice git. Your first exercise is to add your name to this list.
```

Comment:
Yeah i cleaned it up and understood my problem....when i was doing a
checkout...the automatic merge was updating my master with the changes of
the branch....i'm sending another pull request.

On Tue, Mar 3, 2015 at 6:46 PM, Harsh Gupta <notifications@github.com>
wrote:

> In README.md~
> <https://github.com/AGV-IIT-KGP/git-sandbox/pull/39#discussion_r25686084>:
>
> > @@ -0,0 +1,25 @@
> > +This is a place to practice git. Your first exercise is to add your name to this list.
>
> Please remove this file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AGV-IIT-KGP/git-sandbox/pull/39/files#r25686084>.
>